src/components/Shared/Buttons.js

```diff
@@ -233,15 +233,16 @@ export const getButtons = platformLabels => [
     onClick: 'onSaveClear',
     color: '#231f20',
     image: 'saveclear.png',
-    main: true
+    main: true,
+    disabled: 'disabledSavedClear'
   },
   {
     key: 'Submit',
     label: platformLabels.Submit,
     condition: 'isSaved',
     onClick: 'onSave',
     color: '#4eb558',
-    disabled: 'disabledSubmit || isPosted || isClosed',
+    disabled: 'disabledSubmit || isPosted',
```

Comment:
why u remove ||closed? we have records are closed only and not posted so save should be disabled in this case

Reply:
this changes is unintentional